crates/router/src/connector/tsys/transformers.rs

```diff
@@ -464,6 +464,7 @@ impl From<TsysTransactionDetails> for enums::RefundStatus {
     fn from(item: TsysTransactionDetails) -> Self {
         match item.transaction_status {
             TsysTransactionStatus::Approved => Self::Pending,
+            //Connector calls refunds as Void
             TsysTransactionStatus::Void => Self::Success,
```

Comment:
If refund is called as void, what do they call for voiding a payment? Or do they even support?

Reply:
Both Void and Refunds are addressed as `Void`
We handle them accordingly in response on the designated call either payments or refunds